README_ENDUSER.md

```diff
@@ -0,0 +1,8 @@
+# Flight Price Watcher
+
+This applet will help you to find the best flight prices.
+You have to create an account on <https://rapidapi.com/skyscanner/api/skyscanner-flight-search> in order to get your
+API key.
+You will have to get the required API key which name is "X-RapidAPI-Key" and paste it in the designated input.
```

Comment:
This does not explain how the Applet works really. What about the color changes ?